public/scripts/d3/margins.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable */
+
+var width = 1000;
```

Comment:
Width here has global scope

You need to decide how you are going to modularize your code.

Using an iife is the easiest option, happy to go into more detail if you like